scripts/release.sh

```diff
@@ -34,10 +34,8 @@ old_tag=$(grep version Cargo.toml | head -1 | cut -d'"' -f 2)
 sed -i "s/{ version = \"=$old_tag\", path/{ version = \"=$new_tag\", path/g" Cargo.toml
 sed -i "s/version = \"$old_tag\"/version = \"$new_tag\"/g" Cargo.toml
 git add Cargo.toml
-cargo update
```

Comment:
Should keep this line.

Reply:
Remove it, then run `cargo check --workspace` . It breaks the release script, because of some weird things going on with syn. Updates to cargo.lock should pass a lint which runs with a PR anyway, not discovering a broken dep when trying to do a release.

Reply:
Hmm i see...